release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,7 +3322,61 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Issued: 2024-06-19
+
+{product-title} release 4.14.30, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:3881[RHSA-2024:3881] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3918[RHSA-2024:3918] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.30 --pullspecs
+----
+
+[id="ocp-4-14-30-enhancements"]
+==== Enhancements
+
+The following enhancement is included in this z-stream release:
+
+[id="ocp-4-15-30-password-colon-character"]
+===== Allowing pull secret passwords to contain a colon character
+
+This release adds the ability to include a colon character in your password for the Openshift Assisted Installer. (link:https://issues.redhat.com/browse/OCPBUGS-35034[*OCPBUGS-35034*])
+
+[id="ocp-4-15-30-csi-topology-awareness"]
+===== Making Cinder CSI Driver Topology feature configurable
+
+Previously, CSI topology awareness was constantly active. With this release, Cinder CSI topology awareness is based on the compute and storage availability zones and introduces a mechanism for disabling topology functionality. (link:https://issues.redhat.com/browse/OCPBUGS-34792[*OCPBUGS-34792*])
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously, the OpenShift Agent Installer reported installed SATA SDDs as removable and refused to use any of them as installation targets. With this release removable disks are eligible for installation and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
+
+* Previously, an AWS policy issue prevented the Cluster API Provider for AWS from retrieving the necessary domain information. As a consequence, installing an AWS hosted cluster with a custom domain failed. With this update, the policy issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34856[*OCPBUGS-34856*])
+
+* Previously, when a cluster or BareMetal Host (BMH) was deleted, a `PreprovImage` image was created during the powering off before delete stage. As a result, the cluster resource got stuck. With this release, an exception is created for the powering off before delete stage and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34814[*OCPBUGS-34814*]).
+
+* Previously, when `virtualHostedStyle` with the `region` endpoint was set in the image registry operator config, the image registry ignored the virtual hosted style config and failed to start. This release adds a new upstream distribution configuration (force-path style) in favor of the downstream-only version (virtual-hosted style). (link:https://issues.redhat.com/browse/OCPBUGS-34668[*OCPBUGS-34668*])
```

Comment:
```suggestion
* Previously, when you enabled `virtualHostedStyle` with `regionEndpoint` in the Image Registry Operator configuration, the image registry ignored `virtualHostedStyle` and failed to start. This release adds a new upstream distribution configuration in favor of the downstream-only version. (link:https://issues.redhat.com/browse/OCPBUGS-34668[*OCPBUGS-34668*])
```

Reply:
- Is `virtualHostedStyle` a parameter we can specify that in the text.
- I am not sure what this means "This release adds a new upstream distribution configuration in favor of the downstream-only version." 
- Please confirm the values in back ticks. For example, if it's `region` endpoint or `regionEndpoint`